.github/workflows/laravel.yml

```
@@ -2,9 +2,9 @@ name: Laravel
 
 on:
   workflow_dispatch:
-#  push:
-#   pull_request:
-#     branches: [ "1.16/release" ]
+  push:
+   pull_request:
+    branches: [ "1.16/release" ]
```

This should be just the `push` that will get all the conditions

```suggestion
  push:
  # pull_request:
  #  branches: [ "1.16/release" ]
```